AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Template/ConfirmRegistrationTemplate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AzureDay.WebApp.Notification.Email.Ext;
```

Comment:
the whole file is redundant and should be removed, registration will be handled by azure ad b2c